packages/dx-react-scheduler-material-ui/src/templates/views/horizontal/day-scale/cell.jsx

```diff
@@ -22,7 +22,7 @@ const styles = theme => ({
   },
 });
 
-const CellBase = ({
+const CellBase = React.memo(({
```

Comment:
Should we use `React.memo` here? I think it is not a good idea, because we have a lot of cells on views.

And same with other `cells`

Reply:
These (and other) cells are rendered every time an appointment is changed, added or deleted. And they shouldn't